mteb/tasks/Retrieval/law/GerDaLIRRetrieval.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class GerDaLIR(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="GerDaLIR",
+        description="The dataset consists of documents, passages and relevance labels in German.",
```

Comment:
Maybe worth adding some details here on why we call it "Small"?

Reply:
What about this? Feel free to rephrase it.
"The dataset consists of documents, passages and relevance labels in German. In contrast to the original dataset, only documents that have corresponding queries in the query set are chosen to create a smaller corpus for evaluation purposes."